app/views/privacy/privacy-policy.njk

```diff
@@ -78,6 +79,19 @@
     <p class='mpdp-text-color-dark-grey govuk-!-margin-top-8 govuk-!-margin-bottom-0'>This notice was last updated on 22 September 2022. </p>
 
   </div>
+  <div class="govuk-grid-column-one-third">
+    {{relatedContent([
+      {
+        id: "ffCLink",
+        text: "Cookies",
+        url: "https://www.gov.uk/help/cookies"
+      },
+      {
+        id: "ffTCLink",
+        text: "Terms and conditions",
+        url: "https://www.gov.uk/help/terms-conditions"
+      }])}}
+  </div>
```

Comment:
Since this is the first time we are introducing links which are different to rest of the pages, you also need to modify the tests somewhat so you can test that Privacy Policy, search and any other page with related content display the expected links.